Formula/jinja2-cli.rb

```diff
@@ -0,0 +1,60 @@
+class Jinja2Cli < Formula
+  include Language::Python::Virtualenv
+
+  desc "CLI for the Jinja2 templating language (like a fancier 'envsubst')"
+  homepage "https://github.com/mattrobenolt/jinja2-cli"
+  url "https://files.pythonhosted.org/packages/50/24/a774867a93c19d21f132154b509ad014ab22106e1927d0241b556cf8c836/jinja2-cli-0.6.0.tar.gz"
+  sha256 "4b1be17ce8a8f133df02205c3f0d3ebfc3a68e795d26987f846a2316636427b7"
+
+  bottle :unneeded
```

Comment:
Is there a reason we would not bottle this ? Some of the other formulae I've looked at uses bottles so this should most likely be removed.

Reply:
No there isn't. I had this declared in my private repo, but it probably should be bottled when made publicly available. 

I suppose that if I remove this `bottle :unneeded` declaration, then the Brew Test Bot will sort things out (as stated in [the docs](https://github.com/Homebrew/brew/blob/81b6623774089c5eaff9eb52066d46717e56f286/docs/Bottles.md#creation))?

I'll remove the declaration now just in case.